activepieces/activepieces
# feat: Implemented Trello Piece

## What does this PR do?

Implements a 'create card' action piece for Trello. 
Implements a 'create card' action piece for Trello
Implements documentation update effecting:
- Apps.mdx
- [new] trello.mdx
- quickstart piece guide (updated to reflect new import paths)

Fixes #278 (partial fix)

## Type of change




## How should this be tested?

Tested in a codespaces environment, please use the following test flow outlined in the documentation:

Obtain API Key & Token:
1. Go to https://trello.com/app-key
2. Copy 'personal key' and enter it into the Trello API Key connection
3. Click 'generate a token' in trello
4. Copy the token and enter it into the Trello Token connection
5. Your connection should now work!

Creating your first Trello card using Activepiece:
1. Create a new collection
2. Select trigger as 'schedule' and we will use its default value of 5 minutes
3. Click on + to add a new piece
4. Select the 'Trello' piece
5. Select the 'Create Card' Action on the right-hand sidebar
6. Select your Trello API Key connection you created above
7. Select your Trello Token Key connection above
8. Head over to the Trello board you wish to create a new card in
9. Create a new card, lets call it 'test'
10. Click into that card
11. In the URL, add '.json', this should return a wall of JSON text
12. Look for a value called 'idList', it should look something like this: `"idList":"5aea17e911c71fee96375d94"`
13. Copy the idList value, in our case `5aea17e911c71fee96375d94` and enter it into the idList field
14. Set the card name you'd like to create
15. Set a description to enter into the card (optional)
16. Lastly, lets test our flow by clicking 'Test Flow'
17. A new card should have been created in the selected list
18. Now click publish, and thats it!

Fetching a card via Activepieces

1. Create a new collection
2. Select trigger as 'schedule' and we will use its default value of 5 minutes
3. Click on + to add a new piece
4. Select the 'Trello' piece
5. Select the 'Get Card' Action on the right-hand sidebar
6. Select your Trello API Key connection you created above
7. Select your Trello Token Key connection above
8. Head over to the Trello board you wish to create a new card in
9. Create a new card, lets call it 'test'
10. Click into that card
11. In the URL, add '.json', this should return a wall of JSON text
12. Look for a value called 'id', it should look something like this: `"id":"5aea17e911c71fee96375d94"`
13. Copy the idList value, in our case `5aea17e911c71fee96375d94` and enter it into the idList field
14. Lastly, lets test our flow by clicking 'Test Flow'
15. A new card should have been created in the selected list
16. Now click publish, and thats it!
